src/picongpu/include/plugins/radiation/radFormFactor.hpp

```diff
@@ -82,6 +82,29 @@ namespace picongpu
   } // end namespace: radFormFactor_CIC_1Dy
 
 
+  namespace radFormFactor_Gauss_spherical
+  {
+    class radFormFactor
+    {
+    public:
+      HDINLINE radFormFactor(void)
```

Comment:
please remove the constructor because there is no need to define a empty default constructor